src/passes/Inlining.cpp

```diff
@@ -446,6 +449,20 @@ struct Inlining : public Pass {
     // return whether we did any work
     return inlinedUses.size() > 0;
   }
+
+  // Checks if we are allowed to inline source into target.
+  bool canInlineInto(Name target, Name source) {
```

Comment:
Nit: How about `isProfitableToInline` or something..? It is not that technically we cannot inline but we choose not to because of performance reasons..

Reply:
Maybe `isUnderSizeLimit` to be precise?